src/main/java/org/openstreetmap/atlas/utilities/command/Levenshtein.java

```diff
@@ -0,0 +1,50 @@
+package org.openstreetmap.atlas.utilities.command;
+
+/**
+ * @author lcram
+ */
+public final class Levenshtein
+{
+    /**
+     * Compute the Levenshtein distance between two given strings.
+     *
+     * @see "https://en.wikipedia.org/wiki/Levenshtein_distance"
+     * @param string1
+     *            the first string
+     * @param string2
+     *            the second string
+     * @return the distance
+     */
+    public static int levenshtein(final String string1, final String string2)
```

Comment:
Check out `StringUtils.getLevenshteinDistance(string1, string2, distanceThreshold)`. That's what we used in other parts of the code.

Reply:
Man this would have saved so much time! Will check this out.